core/src/main/java/be/orbinson/aem/dictionarytranslator/servlets/action/ReplicateDictionaryServlet.java

```diff
@@ -46,8 +53,14 @@ protected void doPost(SlingHttpServletRequest request, @NotNull SlingHttpServlet
             Resource resource = resourceResolver.getResource(path);
 
             if (resource != null) {
-                DistributionRequest distributionRequest = new SimpleDistributionRequest(DistributionRequestType.ADD, true, path);
-                distributor.distribute("publish", resourceResolver, distributionRequest);
+//                DistributionRequest distributionRequest = new SimpleDistributionRequest(DistributionRequestType.ADD, true, path);
+//                distributor.distribute("publish", resourceResolver, distributionRequest);
+                try {
+                    LOG.info("THe wonderfull path here is: "+path);
+                    replicator.replicate(resourceResolver.adaptTo(Session.class), ReplicationActionType.ACTIVATE, path);
```

Comment:
Did you test this activates correctly because this will not recursively activate/replicate

Reply:
made it recursive